--- a/src/plugins/index.ts
+++ b/src/plugins/index.ts
@@ -22,6 +22,7 @@
 import { voteMap } from './votemap';
 import { voteMapMods } from './votemapmods';
 import { warnPlayers } from './warn-players';
+import { panelBridge } from './panelBridge/panelBridge';
 const plugins = [
   skipmap,
   voteMap,
@@ -46,6 +47,7 @@
   levelSync,
   officialKothDb,
   adminsReloadConfig,
+  panelBridge,
 ];
 
 export const initPlugins = async (id: number) => {
